File: 백준/Silver/2563. 색종이/색종이.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        boolean[][] paper = new boolean[100][100];
        int T = scanner.nextInt();
        
        for (int i = 0; i < T; i++) {
            int x = scanner.nextInt();
            int y = scanner.nextInt();
            
            for (int j = x; j < x + 10; j++) {
                for (int k = y; k < y + 10; k++) {
                    paper[j][k] = true;
                }
            }
        }
        
        int area = 0;
        for (int i = 0; i < 100; i++) {
            for (int j = 0; j < 100; j++) {
                if (paper[i][j] == true) {
                    area++;
                }
            }
        }
        
        System.out.println(area);
    }
}

File: 백준/Silver/2563. 색종이/색종이.kt
import java.util.Scanner

fun main() {
    val scanner = Scanner(System.`in`)
    
    val paper = Array(100) { BooleanArray(100) }
    val T = scanner.nextInt()
    
    for (i in 0 until T) {
        val x = scanner.nextInt()
        val y = scanner.nextInt()
        
        for (j in x until x + 10) {
            for (k in y until y + 10) {
                paper[j][k] = true
            }
        }
    }
    
    var area = 0
    for (i in 0 until 100) {
        for (j in 0 until 100) {
            if (paper[i][j]) {
                area++
            }
        }
    }
    
    println(area)
}
